### PR TITLE
Issue #213 - Fix Smarton/newsletter URL

### PR DESCRIPTION
**Pull request by FrancescoSTL**
R? @chrismore

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -329,10 +329,10 @@ function showSidebar(sidebarProps) {
                         showSearch();
                         break;
                     case 'smarton':
-                        tabs.open('https://www.mozilla.org/teach/smarton/security/');
+                        tabs.open('https://www.mozilla.org/teach/smarton/security/?utm_source=fxonboarding&amp;utm_medium=firefox-browser&amp;utm_campaign=onboardingv1&amp;utm_content=security/');
                         break;
                     case 'newsletter':
-                        tabs.open('https://www.mozilla.org/#newsletter-subscribe');
+                        tabs.open('https://www.mozilla.org/?utm_source=fxonboarding&amp;utm_medium=firefox-browser&amp;utm_campaign=onboardingv1&amp;utm_content=community#newsletter-subscribe');
```
Comment (chrismore) [inline index.js]: @FrancescoSTL the newsletter URL is wrong given #208 do just want to handle the smartone link with this issue first?

Comment (FrancescoSTL) [inline index.js]: @chrismore will do

**Comment by FrancescoSTL** [comment]
@chrismore r?

**Comment by chrismore** [comment]
@FrancescoSTL r+

**Pull request merged**